frontend/reducers/ui/selected_users_ui_reducer.js

```diff
@@ -4,6 +4,8 @@ import { SELECT_USER, SELECT_USERS, UNSELECT_USER } from '../../actions/ui_actio
 import { CLEAR_MODAL } from '../../actions/modal_actions.js';
 import { RECEIVE_CHANNEL } from '../../actions/channel_actions';
 
+// TODO: change selectedUsers data structure to a Set
+
```

Comment:
lets keep master branch clean. don't send up commented out code to it. unless the comments are for documentation purposes.